CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Develop
 
+### Added
+- Added configuration options to set Html5Metadata interface values. (#60)
```

Comment:
We usually try to avoid redundantly wordings and take the heading `added` as the first word of the changelog entry. 

```suggestion
- Configuration options to set `Html5Metadata` interface values.
```

Reply:
I updated this. I preserve `#60` reference because it documents why this change was introduced. If you still think that it shuld be removed, let me know so I remove that.